assets/prometheus-operator/0podmonitor-custom-resource-definition.yaml

```diff
@@ -0,0 +1,235 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I think this might be our fault, because in telemeter we are using a different version of prom-operator. cc @brancz

Reply:
This file needs to be removed as seen [here](https://github.com/openshift/cluster-monitoring-operator/blob/6ce17a90badc8b0d51862ff7c1916689636431db/hack/build-jsonnet.sh#L47) with the other CRDs.

Reply:
Removed in updated commit -> 6e206008bbe5027bd0af6893ed9ec8702e7cb250.